src/sentry/static/sentry/app/components/modals/sentryAppDetailsModal.jsx

```diff
@@ -20,10 +42,35 @@ export default class SentryAppDetailsModal extends React.Component {
     closeModal: PropTypes.func.isRequired,
   };
 
-  render() {
+  getEndpoints() {
+    const {sentryApp} = this.props;
+    return [['featureData', `/sentry-apps/${sentryApp.slug}/features/`]];
+  }
+
+  featureTags(features) {
+    return features.map(feature => {
+      const feat = feature.featureGate.replace(/integrations/g, '');
+      return <StyledTag key={feat}>{feat.replace(/-/g, ' ')}</StyledTag>;
+    });
+  }
+
+  renderBody() {
     const {sentryApp, closeModal, onInstall, isInstalled, organization} = this.props;
+    const {featureData} = this.state;
+    // Prepare the features list
+    const features = (featureData || []).map(f => ({
+      featureGate: f.featureGate,
+      description: (
+        <span dangerouslySetInnerHTML={{__html: singleLineRenderer(f.description)}} />
```

Comment:
@MeredithAnya Is it possible that `f.description` might be `undefined`? I think that might cause a bug here.

Reply:
`f.description` should definitely should always be there